pac4j-jwt/src/main/java/org/pac4j/jwt/credentials/authenticator/JwtAuthenticator.java

```diff
@@ -55,12 +60,16 @@ public JwtAuthenticator(final String secret) {
     public void validate(TokenCredentials credentials) {
         CommonHelper.assertNotBlank("secret", secret);
 
-        final String token = credentials.getToken();
+        String token = credentials.getToken();
         boolean verified = false;
         SignedJWT signedJWT = null;
 
         try {
             // Parse the token
+            if (Base64.isBase64(token)) {
```

Comment:
That depends. If you create the two new methods, there are several side effects. Two of which are:

1. I am forced to call the method on JWTAuthenticator, rather than a generic Authenticator. It would be best if we could keep things separate as much as possible.

I can do:

```
Authenticator a = new JwtAuthenticator()
a.validate()
```

The change would be:

```
JwtAuthenticator a = new JwtAuthenticator()
a.validateToken()

... or ...
a.validateBase64Token()
```


2. If you aim to make things easier for the developer, this should just work. Because otherwise, I'd have declare a dependency in my own project, and make that isBase64 call anyway, which means you will end up with that dependency anyway, and have forced me to call separate API methods. I think the framework should just handle this and protect the developer, but if you are OK with the above concerns, I'll add the methods.